dependencies.yaml

```diff
@@ -115,24 +115,15 @@ dependencies:
               cuda: "11.8"
             packages:
               - nvcc_linux-64=11.8
-              - libcufile=1.4.0.31
-              - libcufile-dev=1.4.0.31
-              - libnvjpeg=11.6.0.55
-              - libnvjpeg-dev=11.6.0.55
           - matrix:
               arch: aarch64
               cuda: "11.8"
             packages:
               - nvcc_linux-aarch64=11.8
-              - libnvjpeg=11.6.0.55
-              - libnvjpeg-dev=11.6.0.55
           - matrix:
               cuda: "12.*"
             packages:
               - cuda-nvcc
-              - libcufile-dev
```

Comment:
This matrix could be included on aarch64, which would pull in libcufile-dev. That isn't what we want. Thankfully we never ran into this in practice, to my knowledge.